CHANGELOG.md

```diff
@@ -0,0 +1,181 @@
+nsxiv
+-----
+
+**Stable releases**
+
+**[v27](https://github.com/nsxiv/nsxiv/archive/v27.tar.gz)**
+*(September 14, 2021)*
+
+Changes:
+  * Re-release under the name nsxiv
+  * Sxiv.foreground and Sxiv.background changed to Nsxiv.window.foreground and Nsxiv.window.background
+  * Rework the build system (#19). Now by default we'll build with only optional dependencies that are already installed
+Added:
+  * Fill scale mode (#2)
+  * Configurable X window title (#23)
```

Comment:
```suggestion
  * Configurable X window title with the `-T` flag (#23)
```
this maintains consistency with with the rest of the changelog as every time an option was added it was mentioned along it's flag.

Reply:
Writing it like this seems like the only option to configure the X window title is with the `-T` flag, but `nsxiv` also support changing this thru `config.h`.